syntax = "proto3";

message RequestMsg
{
    int32 cmdType= 1;
    bytes clientId= 2;
    bytes serverId= 3;
    bytes sign= 4;
    bytes data= 5;
}
message RespondMsg
{
    bool rv=1;
    int32 seckeyid= 2;
    bytes clientId= 3;
    bytes serverId= 4;
    bytes data= 5;
}